### PR TITLE
Update topic-experiences-discovery-curation.md

### PR DESCRIPTION
**Pull request by keremyuceturk**
Updating with some of our newer experiences in Teams and Outlook on the web.

### CODE REVIEW

File: Viva/topics/topic-experiences-discovery-curation.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ The properties are identified from the files and pages that are part of the evid
...
 
-Viva Topics then, when the context is appropriate, will suggest these topics to be highlighted on all SharePoint modern site pages in your tenant. The topic can also be directly referenced on the SharePoint modern site page by a page author. When a user is curious to learn more about a topic, they can select the highlighted topic to view a **Topic summary** card that provides a short description. And if they want to learn more, they can select a **Topic details** link in the summary to open the detailed topic page.
+Viva Topics then, when the context is appropriate, will suggest these topics to be highlighted or displayed on different Microsoft 365 apps, such as SharePoint modern  pages or on Outlook on the web. The topic can also be directly referenced on SharePoint pages by an author, on Microsoft Teams in chat messages, as well as in Outlook on the web while writing emails. When a user is curious to learn more about a topic, they can select a topic to view a **Topic summary** card that provides a short description. And if they want to learn more, they can select a **Topic details** link in the summary to open the detailed topic page.
```
Comment (JohanFreelancer9) [inline Viva/topics/topic-experiences-discovery-curation.md:43]:
```suggestion
Viva Topics then, when the context is appropriate, will suggest these topics to be highlighted or displayed on different Microsoft 365 apps, such as SharePoint modern pages or on Outlook on the web. The topic can also be directly referenced on SharePoint pages by an author, on Microsoft Teams in chat messages, as well as in Outlook on the web while writing emails. When a user is curious to learn more about a topic, they can select a topic to view a **Topic summary** card that provides a short description. And if they want to learn more, they can select a **Topic details** link in the summary to open the detailed topic page.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @keremyuceturk, kindly commit the minor suggested change to remove the extra space. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@ruthholls 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by daisyfell** [comment]
Thanks @keremyuceturk , in the future if you want you can use the private repo to submit your changes so they go through with higher priority https://review.learn.microsoft.com/en-us/office-authoring-guide/getting-started-read-write?branch=main

**Pull request merged**